docs/Project.toml

```
@@ -0,0 +1,7 @@
+[deps]
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+ManifoldsBase = "3362f125-f0bb-47a3-aa74-596ffd7ef2fb"
+
+[compat]
+Documenter = "0.24"
```

```suggestion
Documenter = "0.27"
```
We should probably update this to the newest version.

---

Currently still the push does not seem to work, I have not yet seen what the difference here to ManifoldsDiffEq is, where I “stole” the setup from.

---

There seems to be something wrong with deployment keys: https://github.com/JuliaManifolds/ManifoldsBase.jl/runs/4569802272?check_suite_focus=true

---

Sure, I just do not see what since we have a key in the org (the one ManifoldsDiffEq uses, it does not have its own documenter key), but I am not seeing how those two are different - so it magically works over there, but not here - or something, I just don‘t understand that really.

---

Somehow documenter action for ManifoldDiffEq.jl uses GITHUB_TOKEN instead of DOCUMENTER_KEY which may be why it works there. It seems like every time I need to do something with the documenter deployment I just randomly try different things until it works and I always forget what finally did the trick.

---

Ah I will try that as well then. But at least we seem to follow the same ideas when setting it up ;)

---

I checked – my documenter.yml is completely identical with the one from ManifoldsDiffEx (just the branch is master/main). So what else did you set up there? I thought we would get GitHub Token and Documenter Token from the org? DiffEq has no secrets set up no deploy keys...so what is missing here?

---

Ah! I just saw that at 1800 today it did a first push - so whatever we changed it seems to have worked.

---

Right, it seems like a push succeeded.